data/pages-content/accessibility.md

```diff
@@ -0,0 +1,135 @@
+
+### Quiet Room 
+- If you feel overwhelmed and need a quiet space, you can find the quiet room on the **2nd Floor** at **Room 223 and 224**.
+
+### Childcare 
+Free childminding is available for attendees, you can request for this free onsite service when you register your [ticket](https://ep2023.europython.eu/tickets). If you have questions, please contact support@europython.eu.
+Childcare is on the 2nd Floor at Room 225.
+
+- Childcare is on the **2nd Floor** at **Room 225**.
+
+---
+
+## Sprints at The Rajska Building (RB), Žižkov Campus, VŠE (Prague University of Economics and Business)
+**The Rajska Building (RB), Žižkov Campus, VŠE (Prague University of Economics and Business)** is the home of our sprints this year on 22-23 July.
+
+More information on how to get to the venue via https://ep2023.europython.eu/where#v%C5%A1e---sprints-venue
+
+<img src="https://drive.google.com/uc?export=view&id=18v0oZDAmX1iaZVbGdMcwEmO3wIFMcA6O" alt="Žižkov Campus map"/>
+
+<img src="https://drive.google.com/uc?export=view&id=13AMMkECXQonVkXr_XEFSBtFdbBVSQkas" alt="The Rajska Building map"/>
+
+### Accessibility
+* The University is completely accessible for wheelchair users. 
+* Accessible toilet facilities are both on the first and second floor of the Rajska Building. 
+
+### Quiet Room
+* Located on the **2nd floor of Rajska Building**.
+
+### Childcare:
+* There are **no** childcare or childcare facilities at the sprints venue
+
+---
+
+## Social Events
+### PyLadies Lunche
+[PyLadies Lunch](https://ep2023.europython.eu/pyladies-lunch) will be held at **The Prague Congress Centre (PCC)**. This is also accessible by lifts. 
+
+Location of the lunch will be announced during the conference week.
+
+### PyLadies Social Event Venue
+Venue: [Hospůdka na Hradbách](https://www.prague.eu/en/object/food/304/hospudka-na-hradbach), 10 minutes on foot from Prague Congress Centre ([See directions](https://goo.gl/maps/auBv7hwWaWnvJvH9A))
+
+* The tavern and gardens is an outdoor venue. It’s wheelchair accessible and child-friendly.
+* Please note that some seating spots may require assistance. If you have any requirements, please contact helpdesk@europython.eu.
+* There are **no** baby changing facilities at the venue.
+
+### EuroPython Evening Social Venue: Sacre Coeur
+Information about the Evening Social can be found here: https://ep2023.europython.eu/social-event
+
+Accessibility
+
+* TBC with venue
+
+---
+
+## Accessibility FAQ 
+This FAQ covers accessibility information for EuroPython. The EuroPython committee recognizes that accessibility is a continuous process, both systemic and individual. 
+
+If you have specific questions or accessibility needs not covered here, please email us at **helpdesk@europython.eu**.
+
+#### Q: Allergies
+A: Refrain from wearing scents such as perfumes/colognes, scented lotions, clothing with strong detergent scents, etc. Notify us of preventable environmental allergies when purchasing tickets or by email.
```

Comment:
This is for people with allergies or for the general public who can accommodate people with allergies? I think people with allergies know what they need to do to manage already? Also, since allergy is such a broad spectrum, here it feels a bit narrowly targeting specific areas?

Reply:
The latter, this was taken from AlterConf, and I think it's to highlight to people not to wear really strong perfumes, etc and be conscious of people with allergies.